docs/tutorials/05-openshift-rootless-bud.md

```diff
@@ -415,4 +415,4 @@ registry.fedoraproject.org/fedora   33       71d10e102a30   23 hours ago     191
 sh-5.0$ ls -l output/
 total 4
 -rw-r--r--. 1 build build 288 Jul  8 07:48 update-output.txt
-````
\ No newline at end of file
+````
```

Comment:
I'm not seeing a change here, regardless I think it should be 3 backticks not 4

Reply:
Looks like the previous version of the file didn't end with an end-of-line character.